### ISSUE TITLE
Node.js Foundation Package Maintenance Team Meeting 2019-05-13

### ISSUE THREAD

**Issue by mhdawson** [nodejs/package-maintenance#201]
## Time

**UTC Mon 06-May-2019 19:00 (07:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Mon 06-May-2019 12:00 (12:00 PM) |
| US / Mountain | Mon 06-May-2019 13:00 (01:00 PM) |
| US / Central  | Mon 06-May-2019 14:00 (02:00 PM) |
| US / Eastern  | Mon 06-May-2019 15:00 (03:00 PM) |
| London        | Mon 06-May-2019 20:00 (08:00 PM) |
| Amsterdam     | Mon 06-May-2019 21:00 (09:00 PM) |
| Moscow        | Mon 06-May-2019 22:00 (10:00 PM) |
| Chennai       | Tue 07-May-2019 00:30 (12:30 AM) |
| Hangzhou      | Tue 07-May-2019 03:00 (03:00 AM) |
| Tokyo         | Tue 07-May-2019 04:00 (04:00 AM) |
| Sydney        | Tue 07-May-2019 05:00 (05:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Package%20Maintenance%20Team+Meeting+2019-05-06&iso=20190506T19
* or http://www.wolframalpha.com/input/?i=07PM+UTC%2C+May+06%2C+2019+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1i0c4pvpEd-7u3Jw3y9eJ9c8N_z19G6BqNHkzQ6FyfM8/edit>

## Agenda

Extracted from **package-maintenance-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/package-maintenance

* \[key packages\] A list of packages that do not test in latest LTS [#200](https://github.com/nodejs/package-maintenance/issues/200)
* Session at Node.js collaborator summit [#189](https://github.com/nodejs/package-maintenance/issues/189)
* \[baseline practices\] .npmignore or package.json files [#164](https://github.com/nodejs/package-maintenance/issues/164)
* add deprecate guidelines doc \[draft\] [#150](https://github.com/nodejs/package-maintenance/pull/150)
* Which Problems Node.js OSS maintainers/authors face today? [#113](https://github.com/nodejs/package-maintenance/issues/113)

## Invited

* Package Maintenance team: @nodejs/package-maintenance

## Notes

The agenda comes from issues labelled with `package-maintenance-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: https://zoom.us/j/431077278
* For those who just want to watch: https://www.youtube.com/c/nodejs+foundation/live
* youtube admin page: https://www.youtube.com/my_live_events?filter=scheduled

**Comment by ljharb** [comment]
ping - a bunch of us are waiting in the zoom meeting, but there's no host to start the meeting.

**Comment by craftninja** [comment]
We are headed here for now https://us04web.zoom.us/j/581921842

**Comment by rxmarbles** [comment]
Due to issues w/ communication tools (zoom and no access to nodejs youtube channel for streaming) we are pushing todays meeting. to next week on the 13th. Apologize for the late communiqué and hope to see everyone next week

**Comment by sam-github** [comment]
@mhdawson is on vacation for a few more days, he may have forgotten to arrange an alternate, or thought he could make it and something came up. I might have enough permissions to open the zoom meeting if I'd noticed this earlier, sorry.

**Comment by rxmarbles** [comment]
@sam-github an alternate was arranged just due to the issue above we all agreed to meet next week.

**Comment by sam-github** [comment]
I understand, but since it might be a while before Michael is back online, I wanted to say something about why he may have been missing.

**Comment by Emuentes** [comment]
Thanks for the update @sam-github

@ghinks and I are happy to host.

Is there a process for getting access to use the zoom account that is usually listed on the meetings?

**Comment by mhdawson** [comment]
I'm back so the next meeting should be good. For the last meeting I thought I had @mcollina lined to host up but I must have crossed my wire.

**Comment by Trott** [comment]
(Seems like this should be closed?)

**Comment by mhdawson** [comment]
@Trott I think the intent was to postpone it by a week so it is now targetted for the 13th.

**Comment by jchip** [comment]
waiting on zoom, is this the id: https://zoom.us/j/431077278?

**Comment by mhdawson** [comment]
PR for minutes; https://github.com/nodejs/package-maintenance/pull/213
